purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/PurchasesIntegrationTest.kt

```diff
@@ -91,7 +91,7 @@ class PurchasesIntegrationTest : BasePurchasesIntegrationTest() {
                 onSuccess = { offerings ->
                     assertThat(offerings.current).isNotNull
                     assertThat(offerings.current?.availablePackages?.size).isEqualTo(1)
-                    assertThat(offerings.current?.monthly?.product?.sku).isEqualTo(Constants.productIdToPurchase)
+                    assertThat(offerings.current?.annual?.product?.sku).isEqualTo(Constants.productIdToPurchase)
```

Comment:
I changed the project to a new one that only has an annual sku. The idea behind that is that we plan to have a real purchase of an annual sku so we can do better integration tests